index.html

```diff
@@ -1,19 +1,97 @@
+      <img class='image'
+               height='auto'
+               src="img/billG.jpg"
+               width='300'>
+      <h1 class='name'>Bill Gates - Web Developer</h1>
+      <h3 class='tech'>CEO of Microsoft! Ask about me!</h3>
+      <hr class='hr'>
+      <p class='choose'>Occationally I have time to work on side projects, if you're interesed, let me know by signing up on this website.</p><button><a class='team'
+         href="https://www.microsoft.com/en-us/">Lets Team Up</a></button>
+    </section>
+    <section class="middle">
+      <aside class='content aside-child'>
+        <h2 class='aside-h2'>Listen Up</h2>
+        <p class='content-summary'>cardigan Bushwick banjo bespoke chambray lomo small batch sustainable Schlitz mixtape PBR&B deep v Shoreditch normcore pickled irony Tumblr asymmetrical synth flexitarian sartorial forage yr keytar locavore
+        leggings banh mi Wes Anderson farm-to-table occupy Blue Bottle plaid cliche Pitchfork Kickstarter Banksy Thundercats Marfa direct trade viral quinoa Carles Vice food truck brunch messenger bag Godard sriracha gluten-free Cosby sweater
+        tousled authentic squid wolf swag typewriter try-hard put a bird on it whatever street art craft beer VHS freegan actually drinking vinegar disrupt dreamcatcher hashtag semiotics Helvetica 90's literally Austin gentrify you probably haven't
+        heard of them pug Intelligentsia letterpress jean shorts organic aesthetic umami biodiesel Echo Park bitters Etsy Odd Future fap hoodie raw denim Neutra seitan ethical church-key single-origin coffee heirloom fashion axe tote bag meh cred
+        McSweeney's bicycle rights readymade 8-bit polaroid fanny pack beard pour-over stumptown retro DIY fingerstache skateboard pop-up tattooed iPhone master cleanse ennui American Apparel selfies ugh tofu before they sold out vinyl slow-carb
+        trust fund mumblecore blog meggings wayfarers fixie twee cornhole gastropub post-ironic kogi crucifix cray roof party mustache vegan PBR kale chips chillwave salvia keffiyeh lo-fi High Life photo booth XOXO Brooklyn scenester four loko
+        flannel artisan mlkshk art party +1 Portland selvage kitsch Truffaut YOLO paleo chia next level pork belly narwhal distillery Williamsburg butcher shabby chic Tonx 3 wolf moon Pinterest hella</p><!-- Hipster ipsum added  -->
+      </aside>
+      <aside class='content aside-child'>
+        <h2 class='aside-h2'>Stuff I'm good at</h2>
+        <ul class='un-ordered'>
+          <li class='list'>Disc Golf</li>
+          <li class='list'>Coding up UI/UX</li>
+          <li class='list'>Playing lots of video games excessively</li>
+          <li class='list'>Using bootsrap grid</li>
+          <li class='list'>Taking super long naps at work</li>
+        </ul>
+      </aside>
+    </section>
+    <section class="history">
+      <h2 class='work'>Work history</h2>
+      <table class='table'
+             style="width:100%">
+        <tr class='table tr'>
```

Comment:
you don't need to use class names that are redundant with the element. Just use a class selector and nesting ( like .history {
     tr {
          // styles here
     }
}